localization/l10n/bundle.l10n.json

```diff
@@ -99,14 +99,12 @@
   "Move Up": "Move Up",
   "Move Down": "Move Down",
   "Delete": "Delete",
-  "Error loading designer": "Error loading designer",
```

Comment:
Do you know why this gets removed?

Reply:
Just merged from main, seems like this was removed there.